Packages/Network/Sources/Network/Endpoint/Statuses.swift

```diff
@@ -96,14 +97,29 @@ public struct StatusData: Encodable {
       self.expires_in = expires_in
     }
   }
+    
+  public struct MediaAttribute: Encodable {
```

Comment:
I don't understand why we need this new type. I think you can already do it with the existing pipeline?